templates/filters.php

```diff
@@ -9,20 +9,30 @@
 
 	<?php if ( in_array( 'search', $directory_data['filters']['options'], true ) ) { ?>
 	<div class="wsu-people-filter search">
-		<label><span class="screen-reader-text">Start typing to search</span>
-			<input type="search" value="" placeholder="Type to search" autocomplete="off">
+		<label>
+			<span class="screen-reader-text">Start typing to search</span>
+			<input type="search"
+				   value=""
+				   placeholder="<?php echo esc_html( apply_filters( 'wsuwp_people_search_filter_label', 'Type to search' ) ); ?>"
```

Comment:
`esc_html()` is probably fine, but `esc_attr()` would be most appropriate since this data is being output in an HTML attribute.

Reply:
Ah, I got lazy! Thanks for the catch!